client/app/api/course/Statistics/AssessmentStatistics.ts

```diff
@@ -33,4 +34,12 @@ export default class AssessmentStatisticsAPI extends BaseCourseAPI {
       `${this.#urlPrefix}/${assessmentId}/main_statistics`,
     );
   }
+
+  fetchLiveFeedbackStatistics(
+    assessmentId: string | number,
```

Comment:
why `string | number` instead of just `number`?

Reply:
You are right it should just be number.